.ado/publish.yml

```diff
@@ -13,6 +13,15 @@ trigger:
 
 pr: none
 
+variables:
+  latestStableBranch: 0.62-stable
```

Comment:
So this would need to manually be updated, but it feels awkward that we’d have to update it in all the branches. Or can we move this variable definition to a global that’s controlled from an ADO UI?

Reply:
I now also notice in that RNW PR that there’s a separate file for variables, so this should probably all move there (?).

Reply:
> Or can we move this variable definition to a global that’s controlled from an ADO UI?

Yep, looks like we can set them from [the pipeline settings interface](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=classic%2Cbatch#set-variables-in-pipeline).

Reply:
> I now also notice in that RNW PR that there’s a separate file for variables, so this should probably all move there (?).

Looks like using a separate template is only for variables that need to be shared among other ADO templates, so these definitions do not need to be moved.

Reply:
Yeah, since each publish looks at the copy from its own branch, this might be annoying to need to manage this. Maybe just defining the dist-tag and updating just that would be easier. If you can set variables to use between multiple branches that would work well (haven't tried it, but I think you might be able to).

Re the release variables file, that's actually not there anymore on RNW master but might be coming back. For RNW we dictate dist-tag in the Beachball configuration in each package.json though. When using Beachball to publish a whole repo, you don't want to pass a tag to the CLI, since different packages might have different tagging schemes.

Reply:
> If you can set variables to use between multiple branches that would work well (haven't tried it, but I think you might be able to).

Yeah [the docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#set-variables-in-pipeline) say:

> You can also define variables in the pipeline settings UI (see the Classic tab) and reference them in your YAML.

Reply:
Moved this out of the file 👍